utilities/tools/ConvertTo-ARMTemplate-ConvertChildren.Tests.ps1

```diff
@@ -0,0 +1,80 @@
+﻿Describe 'Convert bicep files to ARM' {
+    BeforeAll {
```

Comment:
Is there a reason why we have 4 different files instead of 1 with just the different describe blocks?

Reply:
Because the BeforeAll block runs the conversion script and actually changes files. Each file uses a different parameter setting for the script, so the result is always differently.